src/pages/Menu/Menu/Menu.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import './Menu.scss';
+import Category from './components/Category';
+import Order from './components/Order';
+import MenuDisplay from './components/MenuDisplay';
+import Top from '../../../components/Top/Top';
+
+const Menu = () => {
+  const [menuDisplay, setMenuDisplay] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/Products.json')
+      .then(response => response.json())
+      .then(data => setMenuDisplay(data));
+  });
+
+  return (
+    <div className="menu">
+      <header className="category">
+        <Category />
+      </header>
+      <div className="filter">
+        <Order />
+      </div>
+      <div className="contentDisplay">
+        {menuDisplay.map(
+          ({
+            id,
+            theme,
+            name,
+            description,
+            servings,
+            cookTime,
+            prepTime,
+            spice,
+            price,
+            totalReviews,
+            image,
+          }) => (
+            <MenuDisplay
+              key={id}
+              theme={theme}
+              name={name}
+              description={description}
+              servings={servings}
+              cookTime={cookTime}
+              prepTime={prepTime}
+              spice={spice}
+              price={price}
+              totalReviews={totalReviews}
+              image={image}
+            />
+          )
+        )}
```

Comment:
```suggestion
        {menuDisplay.map((menu) => (
            <MenuDisplay
              key={id}
              {...menu}
            />
          )
        )}
```

만약 한 객체 안에 있는 모든 property를 그대로 props로 넘기고 싶다면 {...object} 형태로 작성해주면, key={value} 형태로 모든 객츼의 property가 props로 전달되는 JSX 문법이 있습니다.

예시 )
```js
const object = {name:"yeonuk", gender:"male"};
const {name, gender} = object;

<People name={name} gender={gender} />
```
```js
const object = {name:"yeonuk", gender:"male"};

<People {...object} />
```

그래서 지금 작성하신 것과 동일한 결과를 간단하게 할 수 있습니다, 다만, 객체안의 모든 프로퍼티가 다 넘어가게 되므로 반드시 모든 프로퍼티를 넘기고 싶을때만 사용하는게 좋습니다!! 만약 모든 프로퍼티를 넘길 필요가 없는데도 위와 같이 작성하게 된다면 자식 컴포넌트에서 필요 이상으로 많은 props를 알게되므로 좋지 않습니다! 상황에 맞게 적절하게 사용해주세요!

Reply:
영욱님 리뷰 감사합니다!! 코밋 자주하고 리뷰 반영하겠습니다!!

Reply:
이름을 바꿔버리시다니.. 😭

Reply:
이부분 리뷰 반영해주세요!

Reply:
연욱님~ 백엔드에서 받은 데이터 중에 menuDisplay.map()에 적용한 키들은 전부가 아닌데 그래도 말씀해주신 형식으로 바꿔도 되나요?

Reply:
> 

죄송해요ㅜㅜ 오타예요!!!